docs/hub/_toctree.yml

```diff
@@ -42,6 +42,10 @@
       title: Model Card Guidebook
     - local: model-card-landscape-analysis
       title: Landscape Analysis 
+    - local: model-cards-user-studies
```

Comment:
I think this was intendedly removed by @EziOzoani. Is that right? 
(I think adding them is nice, but there was a reason behind it)

Reply:
I just re-added them because the doc-builder was complaining otherwise
I can revert this change

Reply:
oh yeah, oops late to this

it was removed because 3 level indenting wasn't (and possible still isn't) a thing that the toctree could handle (see the many version tries and comments in this [closed PR](https://github.com/huggingface/hub-docs/pull/545))

but to fix the doc-build, adding makes sense

Reply:
Btw as per the config it should not fail :D https://github.com/huggingface/hub-docs/blob/main/docs/hub/_config.py